Bottom: 34aea2338028a7b95d4f4ed946e0896de4655319
Top:    12fdf0386be4f4a3561e2f75c4b4bff50c3d262f
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-07 22:51:54 +0100

Refresh of hid-gtk-remove-hard-coded-keyb

---

diff --git a/src/hid/gtk/gui-misc.c b/src/hid/gtk/gui-misc.c
index 730db5e..745e8ea 100644
--- a/src/hid/gtk/gui-misc.c
+++ b/src/hid/gtk/gui-misc.c
@@ -312,54 +312,6 @@ loop_key_press_cb (GtkWidget * drawing_area, GdkEventKey * kev,
    */
   switch (ksym)
     {
-    case GDK_Up:
-      if (mk == CONTROL_PRESSED)
-	{
-	  hid_actionl ("Display", "Scroll", "8", NULL);
-	  hid_actionl ("Display", "Scroll", "0", NULL);
-	}
-      else if (mk == SHIFT_PRESSED)
-	hid_actionl ("MovePointer", "0", "-10", NULL);
-      else if (mk == NONE_PRESSED)
-	hid_actionl ("MovePointer", "0", "-1", NULL);
-      break;
-
-    case GDK_Down:
-      if (mk == CONTROL_PRESSED)
-	{
-	  hid_actionl ("Display", "Scroll", "2", NULL);
-	  hid_actionl ("Display", "Scroll", "0", NULL);
-	}
-      else if (mk == SHIFT_PRESSED)
-	hid_actionl ("MovePointer", "0", "10", NULL);
-      else if (mk == NONE_PRESSED)
-	hid_actionl ("MovePointer", "0", "1", NULL);
-      break;
-
-    case GDK_Left:
-      if (mk == CONTROL_PRESSED)
-	{
-	  hid_actionl ("Display", "Scroll", "4", NULL);
-	  hid_actionl ("Display", "Scroll", "0", NULL);
-	}
-      else if (mk == SHIFT_PRESSED)
-	hid_actionl ("MovePointer", "-10", "0", NULL);
-      else if (mk == NONE_PRESSED)
-	hid_actionl ("MovePointer", "-1", "0", NULL);
-      break;
-
-    case GDK_Right:
-      if (mk == CONTROL_PRESSED)
-	{
-	  hid_actionl ("Display", "Scroll", "6", NULL);
-	  hid_actionl ("Display", "Scroll", "0", NULL);
-	}
-      else if (mk == SHIFT_PRESSED)
-	hid_actionl ("MovePointer", "10", "0", NULL);
-      else if (mk == NONE_PRESSED)
-	hid_actionl ("MovePointer", "1", "0", NULL);
-      break;
-
     case GDK_Return:		/* Accept cursor location */
       if (g_main_loop_is_running (*loop))
 	g_main_loop_quit (*loop);
